_docs/examples/ios.md

```diff
@@ -112,21 +114,24 @@ Memory.readUtf8String(data.bytes(), data.length());
 2. Converting NSData to Binary Data
 -
 ```
+var data = new ObjC.Object(args[2]);
 Memory.readByteArray(data.bytes(), data.length());
 ```
 
 3. Iterating an NSArray
 -
 ```
-var count = array.count();
-for (var i = 0; i !== count; i++) {
+var array = new ObjC.Object(args[2]);
+var count = array.count().valueOf();
+for (var i = 0; i < count; i++) {
```

Comment:
Would be good to use `!==` here. This is so off-by-one errors result in an infinite loop rather than a subtle bug that might go unnoticed for a long time.

Reply:
We synced up off-bug, but this makes all the sense in the world to do!